webpack.config.js

```diff
@@ -52,7 +52,7 @@ const defaultBabelLoader = {
   options: {}
 }
 
-// Browsers
+// Node bundled umd file, with native async await calls
 const browserBundle = copy(baseBundleConfig)
```

Comment:
If this is the node bundle, shall we rename this variable to not add extra confusion?

Reply:
@TimBeyer Good catch - fixed it!

Reply:
btw - it's the browser bundle, must have mixed it up when writing that comment